eth/vm/state.py

```diff
@@ -218,7 +205,7 @@ def revert(self, snapshot: Tuple[Hash32, UUID]) -> None:
         # now roll the underlying database back
         self._account_db.discard(account_snapshot)
 
-    def commit(self, snapshot: Tuple[Hash32, UUID]) -> None:
+    def commit(self, snapshot: Tuple[Hash32, JournalDBCheckpoint]) -> None:
```

Comment:
I'm curious, is this change because `mypy` `0.720` started noticing that this can not be `UUID` anymore? If so, hooray for improved type safety :)

Reply:
Yes, new errors showed up when I upgraded `mypy`